build.bat

```diff
@@ -13,7 +13,7 @@ SET BUILD_DIR=%~d0%~p0%
 SET NANT="%BUILD_DIR%lib\Nant\nant.exe"
 SET build.config.settings="%DIR%\.uppercut"
 
-%NANT% /logger:"NAnt.Core.DefaultLogger" /quiet /nologo /f:"%BUILD_DIR%.build\default.build" -D:build.config.settings=%build.config.settings% %*
+%NANT% /logger:"NAnt.Core.DefaultLogger" /quiet /nologo /f:"%BUILD_DIR%.build\default.build" -D:build.config.settings=%build.config.settings% -D:msbuild.configuration="Release" %*

```

Comment:
Is this not the default?

Reply:
It is, but since I am using the value in an if statement early in the build process, it didn’t seem to have been set yet, and the only solution I could see was to set it in the build.bat file.

Reply:
If I didn't make this change, I get the following:

![image](https://user-images.githubusercontent.com/1271146/67844158-2f5bfd00-faf5-11e9-93cc-81a08526a1c3.png)

Reply:
Ah, fair!